blibli-backend-framework-newrelic/src/main/java/com/blibli/oss/backend/newrelic/configuration/EnableNewRelicReactorInstrumentation.java

```diff
@@ -0,0 +1,15 @@
+package com.blibli.oss.backend.newrelic.configuration;
+
+import com.blibli.oss.backend.newrelic.configuration.reporter.NewRelicMongoReporterAutoConfiguration;
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Import({NewRelicReactorInstrumentationAutoConfiguration.class, NewRelicMongoReporterAutoConfiguration.class})
+public @interface EnableNewRelicReactorInstrumentation {
```

Comment:
we need to automatically register the configuration, don't manual using annotation

Reply:
updated to auto configure the module